### ISSUE TITLE
Super Mario™ 3D All-Stars updates do not work

### ISSUE THREAD

**Issue by ArchangeGabriel** [yuzu-emu/yuzu#8217]
Thanks to the great work by @liamwhite, we can now run those games. :) But… all updated versions do not work, I had to disable them in order for the game to boot.

This is on a Steam Deck, but on Discord people reported the same issue on Windows as well.
Here is the [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/8501161/yuzu_log.txt).

As one can easily notice, they are thousands of `HW.Memory <Error> core/memory.cpp:operator():497: Unmapped Read32`.

**Comment by baggyg** [comment]
How does one go about disabling the updates?

**Comment by luisalvarado** [comment]
Hi Yuzu developers, I created a video showing this game working. I am new to the emulator (only 2 days) but found this particular one working (My son wanted to play). The proof of concept is here https://www.youtube.com/watch?v=v8o9S-pmvvQ

The only issue was the sound sometimes it sounds or cuts a bit and that enabling SMAA would make the game lag. Other than that, it was playable and perfect.

**Comment by ArchangeGabriel** [comment]
@luisalvarado What version did you use? This issue is about updates not working, the base game itself works.

**Comment by luisalvarado** [comment]
@ArchangeGabriel 1279 it said. I am using the flatpak one. Updates to the game?

**Comment by ArchangeGabriel** [comment]
The page is long outdated. And yes, it’s about updates to the game.

**Comment by luisalvarado** [comment]
Confirmed, after 12039812039 hours looking for the 1.1.1 Update, I applied that. The crackling sound issue was gone but the moment I started any of the games it crashed with 

yuzu[12400]: segfault at 4000000408 ip 00007fe18a4cfe38 sp 00007ffebf3971e0 error 4 in libQt5Core.so.5.15.7[7fe18a29b000+314000]

**Comment by v1993** [comment]
This is a general issue with multi-title games. In addition to updates not working:

* DLCs don't work either
* RomFS dumps only contain main title
* Launch parameters aren't passed properly (?)

**Comment by luisalvarado** [comment]
Hi @v1993 I found this emulator called ryujinx from https://ryujinx.org/download and did the same tests. So the moment I applied the updates, the 3 games worked with the updated version. Dont know if that helps. There was no weird sound either, like a crackling. I think all 3 games were not even lagging. All I did was setup the firmware thing, the prod.key or keys I think it is called and then proceeded to update the game. I also updated breath of the wild. Works perfectly. Have not tested for a longer time, just tested 5 to 10 minutes each game with the 1.1.1 update applied to them. Hope this helps.

**Comment by DeathWrench** [comment]
Sunshine runs fine for me with 1.1.1, my only problem now is that there is seemingly no way of using the home button to invert the controls. If anyone knows how to do this. Please let me know.

**Comment by RinMaru** [comment]
Update 1.1.1 is still broke

**Comment by liamwhite** [comment]
No it's not, it just doesn't display in the title bar. The update is applied just fine

**Comment by RinMaru** [comment]
> No it's not, it just doesn't display in the title bar. The update is applied just fine

it is because the game refuses to launch otherwise

**Comment by liamwhite** [comment]
It launches fine here.

**Comment by RinMaru** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12569932/yuzu_log.txt)

**Comment by mdjima77** [comment]
Hi, all! 

Sorry for resurrecting an old thread, but I joined specifically for this - to share that the game (all 3) **_and_** the 1.1.1 update works perfectly (no glitches in either audio, video, frame rates, etc.) on Yuzu 1713, using a Nintendo Rock Candy wired controller. 

The only thing we (i.e., my 7-yr old) tweaked was activating the controller from the Controls menu (Yuzu / Emulator / Configure / Controls) - this is _the only thing_ we had to do to get past the Intro screen.

**OS:** Windows 10 Home (not activated) 22H2, build 19045.3996
**Hardware:** Lenovo ThinkBook 64bit, AMD Ryzen 7 4700U with Radeon Graphics 2.00 GHz, RAM 16 GB,
